runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1449,7 +1452,8 @@ NNFW_STATUS nnfw_session::train_run(bool update_weights)
   {
     if (update_weights)
     {
-      _execution->train(_training_step++);
+      auto &training_step = _train_info->epoch();
```

Comment:
`training_step` and `epoch` are different. Why did you merge them?

Reply:
There is no need to keep the `epoch` number in `TrainingInfo` because the epoch information depends on the user's argument, not related to the model.

And the `training_step` is located directly in the `Session` object. However, we introduced the `TrainingInfo` to session object. This variable can be merged into the `TrainingInfo` to manage information related to training in one place.

So I tried to merge `training_step` into the `TrainingInfo` while changing the name to `epoch`.

How about merging the `training_step` variable to the `TrainingInfo` while keeping the same name?

Reply:
The name `training_step` looks better.

Reply:
> The name `training_step` looks better.

This code is outdated. Please see the new patch. :)